tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Utility/Logger/AzureLogger.cs

```diff
@@ -29,6 +29,7 @@ public class AzureLogger
             tableEntityObj.MethodName = methodName;
             string date = DateTime.Now.ToUniversalTime().ToString(logTables.AzureRowKeyDateFormat, CultureInfo.InvariantCulture);
             tableEntityObj.RowKey = string.Format(CultureInfo.InvariantCulture, "{0} - {1}", date, Guid.NewGuid().ToString());
+            tableEntityObj.Stacktrace = exception.StackTrace;
```

Comment:
Added stack trace column to the azure log storage so that it will be helpful to pin point the problem